Sources/Workspace/Destination.swift

```diff
@@ -178,6 +178,24 @@ public struct Destination: Encodable, Equatable {
     }
     /// Cache storage for sdk platform path.
     private static var _sdkPlatformFrameworkPath: (fwk: AbsolutePath, lib: AbsolutePath)? = nil
+
+    /// Returns a default destination of a given target environment
+    public static func targetDefaultDestination(target: Triple, hostDestination: Destination) -> Destination? {
```

Comment:
`defaultDestination(for triple: Triple, hostDestination: Destination) -> Destination? {`